src/utils/generateMeasurementTypeText.js

```diff
@@ -0,0 +1,33 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2021 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+/**
+ * Takes measurement type and produces formatted string.
+ * @param measurementType {string}
+ * @returns {string}
+ */
+const generateMeasurementTypeText = (measurementType) => {
+  switch (measurementType) {
+    case "INSTANT":
+      return "Point in Time";
+    case "AVERAGE":
+      return "Average";
+    default:
```

Comment:
I see in the test cases that you have a specific case for `DELTA` but are letting it `default` here. I see in the linked ticket that there are two separate DELTA values that are valid incoming values but we are simply omitting this text in those cases. I wonder, though, if it would be a good idea to explicitly list them as cases here both 1) to better signal expected values, and 2) so it's easier to update in case we want to add measurement type text support for them in the future?

Reply:
Good call!